src/components/Filtration/Filtration.tsx

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import { useSearchParams, Link } from 'react-router-dom';
+import {
+  useEffect,
+  useState,
+} from 'react';
+import cn from 'classnames';
+import { Pagination } from '../Pagination';
+import './Filtration.scss';
+import { getSearchString } from '../../helpers/getSearchString';
+
+type Props = {
+  total: number;
+  children: React.ReactNode;
+};
+
+const sortOptions: { [key: string]: SortType } = {
+  Newest: 'age',
+  Alphabetically: 'name',
+  Cheapest: 'price',
+  Expensive: '-price',
+};
+
+const perPageOptions: PerPageType[] = ['4', '8', '16', 'all'];
+
+export const Filtration: React.FC<Props> = ({ children, total }) => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const [sort, setSort] = useState('Newest');
```

Comment:
it is strange to use string as initial value when you have `SortType` enum and `sortOptions` object for it